lib/model/Attribute.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:openfoodfacts/interface/JsonObject.dart';
+
+class Attribute extends JsonObject {
+  Attribute({
+    this.id,
+    this.name,
+    this.title,
+    this.iconUrl,
+    this.defaultF,
+    this.settingNote,
+    this.settingName,
+    this.description,
+    this.shortDescription,
+    this.match,
+    this.status,
+  });
+
+  factory Attribute.fromJson(dynamic json) => Attribute(
+        id: json[_JSON_TAG_ID] as String,
+        name: json[_JSON_TAG_NAME] as String,
+        title: json[_JSON_TAG_TITLE] as String,
+        iconUrl: json[_JSON_TAG_ICON_URL] as String,
+        defaultF: json[_JSON_TAG_DEFAULT] as String,
+        settingNote: json[_JSON_TAG_SETTING_NOTE] as String,
+        settingName: json[_JSON_TAG_SETTING_NAME] as String,
+        description: json[_JSON_TAG_DESCRIPTION] as String,
+        shortDescription: json[_JSON_TAG_DESCRIPTION_SHORT] as String,
+        match: JsonObject.parseDouble(json[_JSON_TAG_MATCH]),
+        status: json[_JSON_TAG_STATUS] as String,
+      );
+
+  @override
+  Map<String, dynamic> toJson() => JsonObject.removeNullEntries({
+        _JSON_TAG_ID: id,
+        _JSON_TAG_NAME: name,
+        _JSON_TAG_TITLE: title,
+        _JSON_TAG_ICON_URL: iconUrl,
+        _JSON_TAG_DEFAULT: defaultF,
+        _JSON_TAG_SETTING_NOTE: settingNote,
+        _JSON_TAG_SETTING_NAME: settingName,
+        _JSON_TAG_DESCRIPTION: description,
+        _JSON_TAG_DESCRIPTION_SHORT: shortDescription,
```

Comment:
I know it was already like this before, but can we change shortDescription to descriptionShort, so that the field names matches the ones we have in the JSON API? The less surprises / differences, the better. :)

Reply:
`json_serializable` has built-in support for removing field that are `null`, e.g. `@JsonKey(name: 'product_name', includeIfNull: false)`. Like I commented on the PR, I'm worried about the manual implementations of the JSON conversion functions.